press/api/bench.py

```diff
@@ -615,6 +615,24 @@ def update(name, bench):
 	frappe.get_doc("Bench", bench).update_all_sites()
 
 
+@frappe.whitelist()
```

Comment:
Instead of writing it again just use press.api.bench.update ?

Maybe do something like. This should be enough

```python
benches = frappe.get_all("Bench", {"group": name})
for bench in benches:
    frappe.get_cached_doc("Bench", bench).update_all_sites()
```